TWLight/applications/management/commands/notify_applicants_tou_changes.py

```diff
@@ -49,10 +49,11 @@ def handle(self, *args, **options):
                     content_object=app,
                     site_id=site_id(),
                     user=twl_team,
+                    # fmt: off
                     # Translators: This comment is added to pending applications when our terms of use change.
-                    comment=_(
-                        "Our terms of use have changed. "
+                    comment=_("Our terms of use have changed. "
                         "Your applications will not be processed until you log in and agree to our updated terms."
                     ),
```

Comment:
closing brackets for `_()` should be lifted to the same line as the source string